drivers/sensor/icm20948/Kconfig

```diff
@@ -0,0 +1,90 @@
+# Kconfig.ICM20498 - ICM20498 ICM-20498 I2C-based or SPI-based IMU
+
+#
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+menuconfig ICM20948
+	bool "ICM20498 sensor"
+	depends on I2C || SPI
+	default	n
```

Comment:
`default n` is redundant. See the [Kconfig - Tips and Best Practices](https://docs.zephyrproject.org/latest/tools/kconfig/index.html#redundant-defaults) page.